### DISCUSSION TITLE
"Android theme issue: PlatformException during Stripe initialization"

### DISCUSSION THREAD

**Discussion by brief69** [flutter-stripe/flutter_stripe#1884] [Q&A]
Hello,

I'm encountering an issue with Stripe initialization on Android. The app works fine on iOS, but on Android, I'm getting the following error:
PlatformException(flutter_stripe initialization failed, The plugin failed to initialize:
Your theme isn't set to use Theme.AppCompat or Theme.MaterialComponents.
Please make sure you follow all the steps detailed inside the README: https://github.com/flutter-stripe/flutter_stripe#android
If you continue to have trouble, follow this discussion to get some support https://github.com/flutter-stripe/flutter_stripe/discussions/538, null, null)

I've already tried the following:

1. Modified `android/app/src/main/res/values/styles.xml` to use `Theme.MaterialComponents.Light.NoActionBar`
2. Added `com.google.android.material:material:1.5.0` to `android/app/build.gradle`
3. Cleaned and rebuilt the project
4. Verified `AndroidManifest.xml` is using `@style/LaunchTheme`
5. Updated Android Gradle plugin to 8.1.4 and compileSdkVersion to 34
6. Updated flutter_stripe to the latest version (11.0.0)

Despite these attempts, the issue persists. Could you please provide any additional guidance on resolving this theme-related initialization problem?

Thank you for your help!

**Comment by jonyoung123** [comment]
hi, if you haven't fixed it, check this out.

https://github.com/flutter-stripe/flutter_stripe/discussions/538#discussioncomment-2645392
